### DISCUSSION TITLE
nextprev user.php and pm using legacy code?

### DISCUSSION THREAD

**Discussion by Alex-e107nl** [e107inc/e107#5137] [General]
Hi, on e107.nl user.php and pm are using code:
```
// Use NEW nextprev method
		$np_parm['template'] = "<span class='form-group form-inline'>[PREV]&nbsp;&nbsp;[DROPDOWN]&nbsp;&nbsp;[NEXT]<span>";
		$np_parms['prev'] = '&nbsp;&nbsp;&lt;&lt;&nbsp;&nbsp;';
		$np_parms['next'] = '&nbsp;&nbsp;&gt;&gt;&nbsp;&nbsp;';
		$np_parms['np_class'] = 'tbox npbutton btn btn-default';
		$np_parms['dropdown_class'] = 'tbox npdropdown form-control';
		$caption = ''; // legacy has no caption.
```

from 107_core/shortcodes/single/nextprev.php is that right?

![nextprev-pm_2023-12-05](https://github.com/e107inc/e107/assets/11962513/4f54e857-8de0-4fee-9a8f-39138184a450)
![nextprev-user_php_2023-12-05](https://github.com/e107inc/e107/assets/11962513/406f53e1-448c-4e31-81b8-1ec818d42d65)

I had to extend the style.css with:

```
.form-inline {
	position: relative;
	display: inline-flex;
    margin-bottom: 0;
    vertical-align: middle;
	border-radius: var(--bs-border-radius);
	
  }
```

To get it a little better but it should use the nextprev_template.php i think?

![nextprev-pm_2023-12-05_2](https://github.com/e107inc/e107/assets/11962513/662abc59-e6e3-4fc1-9c53-aeb9c4448c24)
![nextprev-user_php_2023-12-05_02](https://github.com/e107inc/e107/assets/11962513/59434405-1347-4b1a-8228-2ec81c861688)

**Comment by Jimmi08** [comment]
Maybe related: 
https://github.com/e107inc/e107/issues/4663

**Comment by Jimmi08** [comment]
Maybe related:

https://github.com/e107inc/e107/issues/4685

**Comment by Jimmi08** [comment]
this issue should be checked too, it is possible that it is solved already

https://github.com/e107inc/e107/issues/4253

EDIT: still issue

**Comment by Alex-e107nl** [comment]
Ai that's a lot, i was surprised with this i thought editing the nextprev_template.php would do the trick ai....

**Comment by Jimmi08** [comment]
@Alex-e107nl your css fix can break other form-inline on other page. If something, you should use
.account-details  .next-prev .form-inline {}  
to apply on for user page. But there is definitely something wrong

![image](https://github.com/e107inc/e107/assets/5429548/98151cee-e669-40b4-8ecd-2e235dbd6077)

**Reply by Jimmi08** [reply]
https://github.com/e107inc/e107/issues/5139

**Comment by Alex-e107nl** [comment]
Thank you @Jimmi08  I have adjusted where it appears in user.php and the pm_template.php!
